templates/cisco_ios_show_ip_cef.textfsm

```diff
@@ -0,0 +1,43 @@
+#
+# This template is only compatible with the following command
+#   - show ip cef [vrf NAME]
+# This is enforced by INDEX file
+# Check cisco_ios_show_ip_cef_detail.textfsm for more details
+#
+Value Required PREFIX ((?:\d{1,3}\.){3}\d{1,3}\/\d{1,2})
+Value TYPE ((?:no\s)?(?:[a-z]+))
+# To support ECMP, NEXTHOP and INTERFACE are defined as lists
+# When no ECMP is available, these will be single item lists
+# However, if there are multiple paths available
+#     then expect to see one item per route/path
+Value List NEXTHOP ((?:\d{1,3}\.){3}\d{1,3})
+Value List INTERFACE ([A-Za-z][A-Za-z0-9\.\/-]+)
+
+Start
+  # >>> Parse EXCEPTIONS
+  # CEF is not enabled
+  ^%IPv4\s+CEF\s+not\s+running$$ -> End
+  # Invalid prefix
+  ^\s+Invalid\s+prefix/mask -> End
+  #
+  # >>> Parse HEADING
+  ^Prefix\s+Next\s+Hop\s+Interface$$ -> Entries
```

Comment:
Do you minding adding the ending lines to the `Start` state as well?

```
  ^\s*$$
  ^. -> Error
```

Reply:
I will do